### PR TITLE
Present speeches to publishing api

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/J9H6FRK1/611-speech-migration-2-implement-publishing-of-format-to-publishing-api

Presenters for Speech format.

### CODE REVIEW

File: app/presenters/publishing_api/speech_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    def image_payload
+      {
+        image: {
+          alt_text: speaker.name,
+          url: speaker.image.url,
+        }
+      }
+    end
+
+    def links_for_policies
```
Comment (gpeng) [inline app/presenters/publishing_api/speech_presenter.rb]: I know we had a conversation about this but it looks like everything else is using `related_policies` for this, passed as a key to the `LinksPresenter`

Comment (steventux) [inline app/presenters/publishing_api/speech_presenter.rb]: There's actually quite a bit of variation in the content schemas, speech schema and others use the key `policies` https://github.com/alphagov/govuk-content-schemas/blob/master/formats/speech/publisher/links.json#L11 see https://trello.com/c/rXXuFPfS/848-rename-publications-related-policies-links-key for the detail on the differences. Happy to change it but conversations I had with Finding Things team yesterday pointed to retaining the `policies` key and removing the `related_policies` one.

**Comment by steventux** [comment]
Thanks for the review @gpeng updated

File: app/presenters/publishing_api/speech_presenter.rb (hunk 1 above)
Comment (gpeng) [inline app/presenters/publishing_api/speech_presenter.rb]: Ok. Fair enough.

**Pull request merged**